frontend/components/product_markup.js

```diff
@@ -0,0 +1,46 @@
+/*
+<div key={ i } className='products-summary'>
+    <div className='products-summary__name'>
+    { datum.name }
+    </div>
+    <div className='products-summary__brand'>
+    <i className='products-summary__brand-by italic'> by </i> { datum.brand }
+    </div>
+    <div className='products-summary__data-input-status'>
+    <i className='italic'> Status </i> { datum.dataInputStatus }
+    </div>
+</div>
+*/
+
+import React, { PureComponent } from 'react';
+
+export default class ProductMarkup extends PureComponent {
+  productName = null;
```

Comment:
why are these variables declared? are you using `this.productName` anywhere? keep in mind, instance variables dont cause a component to re-render, only props/state trigger a re-render when they change